trolley.gemspec

```
@@ -4,15 +4,22 @@ Gem::Specification.new do |s|
   s.name = 'trolley'
   s.summary = 'Trolley Ruby SDK'
   s.description = 'Ruby SDK for interacting with the Trolley API'
-  s.version = '1.0.1'
-  s.homepage = 'https://trolley.com/'
+  s.version = '1.0.2'
+  s.homepage = "https://github.com/Trolley/ruby-sdk"
   s.email = ['developer-tools@trolley.com']
   s.license = 'MIT'
-  s.author = 'Trolley'
+  s.authors = ['Trolley', 'Emi GM']
   s.files = Dir.glob ['README.rdoc', 'LICENSE', 'lib/**/*.{rb,crt}', 'spec/**/*', '*.gemspec']
   s.required_ruby_version = '>= 2.7'
   s.add_development_dependency 'dotenv', '~> 2'
   s.add_development_dependency 'rake', '~> 13'
   s.add_development_dependency 'rubocop', '~> 1'
   s.add_development_dependency 'test-unit', '~> 3'
+  s.metadata = {
+    "bug_tracker_uri"   => "https://github.com/trolley/ruby-sdk/issues",
+    "changelog_uri"     => "https://github.com/PaymentRails/ruby-sdk/releases",
```

This one needs to get updated too
```suggestion
    "changelog_uri"     => "https://github.com/trolley/ruby-sdk/releases",
```